Ports/AvailablePorts.md

```diff
@@ -68,6 +68,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`halflife`](halflife/)                | Half-Life                                                       | 1.0.0                    | https://github.com/SerenityPorts/xash3d-fwgs                                   |
 | [`harfbuzz`](harfbuzz/)                | HarfBuzz                                                        | 2.8.1                    | https://github.com/harfbuzz/harfbuzz                                           |
 | [`hatari`](hatari/)                    | Atari ST/STE/TT/Falcon emulator                                 | 2.4.0-devel              | https://hatari.tuxfamily.org/                                                  |
+| [`imagemagick`](imagemagick/)          | ImageMagick                                                     | 7.1.0-23                 | https://imagemagick.org                                                        |
```

Comment:
This should be squashed into the previous commit

Reply:
Thank you for telling me. I've done that now.